_data/simple-icons.json

```diff
@@ -5625,6 +5625,11 @@
             "hex": "F7B500",
             "source": "https://www.sketch.com/press/"
         },
+        {
+            "title": "Sketchfab",
+            "hex": "1CAAD9",
```

Comment:
Where have you taken the color from?

Reply:
In the press kit page (https://sketchfab.com/press) I downloaded the first image and I got the color from that.

Reply:
Ah okay! Great, then this is good, but please link to this site then instead of root. Thanks!